jest.config.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({ path: '.env.local' });
```

Comment:
What is this needed for?

Reply:
This allows jest to get access to .env.local variables when testing.

Reply:
Do we need environment variables for testing? It doesn't make sense to me that unit tests would need them and I don't see them being used in the code.